src/app/components/mnoe-api/app-instances.svc.coffee

```diff
@@ -62,6 +62,26 @@ angular.module 'mnoEnterpriseAngular'
       Array.prototype.push.apply(_self.appInstances, arr)
       return _self.appInstances
 
+    @getForm = (instance) ->
+      MnoeApiSvc.one('organizations', MnoeOrganizations.selectedId).one('/app_instances', instance.id).one('/setup_form').get()
+
+    @submitForm = (instance, model) ->
+      body = {}
+      for key of model
+        if model.hasOwnProperty(key)
+          body[key] = model[key]
+      MnoeApiSvc.one('organizations', MnoeOrganizations.selectedId).one('/app_instances', instance.id).post('/create_omniauth', body)
+
+    @getSyncs = (instance) ->
+      MnoeApiSvc.one('organizations', MnoeOrganizations.selectedId).one('/app_instances', instance.id).one('/sync_history').get()
```

Comment:
Do not use `one` without id parameter. You should use `customGET` or `doGET`.
cf. https://github.com/mgonto/restangular#custom-methods